src/routing/RouteSelector.jsx

```diff
@@ -90,6 +91,12 @@ export const RouteSelector = () => (
         </SiteLaunchProvider>
       </ProtectedRouteWithProps>
 
+      <ProtectedRouteWithProps path="/sites/:siteName/siteLaunchPad">
```

Comment:
shouldn't this be behind a feature flag since we will only launch this later? For now can manually have a featureFlag.ts which exports allowSiteLaunch=false

Reply:
hey it feels weird to have biz logic there, and there is already a util file `shouldUseSiteLaunchFeature`. Would you be ok with `<SiteLaunchPad>` taking care of this responsibility, and showing a text like `Site Launch Feature is not enabled for this site`?  Since this will involve creating a another story + asking Nat for UI, I am tempted to tackle it in a separate PR. If ok, Ill go ahead to raise a issue for this

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/1324
@harishv7 fixed here